packages/reports/tests/acceptance/report-test.js

```diff
@@ -782,8 +782,8 @@ module('Acceptance | Navi Report', function(hooks) {
       .split('/export?reportModel=')[1];
     await CompressionService.decompressModel(encodedModel).then(model => {
       assert.equal(
-        get(model, 'visualization.metadata.showTotals.grandTotal'),
-        true,
+        get(model, 'visualization.metadata.showTotals.grandTotal.target.value'),
+        'on',
```

Comment:
Looks like we are breaking the model contract here. We should stick to boolean values.

Reply:
.grandTotal pointed to a mouse event rather than a boolean, and I dont believe I changed that. Any advice?